viewer/apiHunts.js

```diff
@@ -781,7 +781,8 @@ ${Config.arkimeWebURL()}sessions?expression=huntId==${huntId}&stopTime=${hunt.qu
         stopTime: req.body.query.stopTime,
         view: req.body.query.view
       },
-      description: req.body.description
+      description: req.body.description,
+      roles: req.body.roles
```

Comment:
should we validate roles exist?

Reply:
Here, I was going to allow roles to be undefined (since old hunts will have undefined roles and the client handles this). Optionally, I could set this to an empty array if there are no roles defined upon creation by the client. I'm not sure which is best